nodesharing-lib/src/main/java/com/redhat/jenkins/nodesharing/ConfigRepo.java

```diff
@@ -232,7 +235,7 @@ private void fetchChanges(@Nonnull TaskLog taskLog) throws InterruptedException,
                 continue;
             }
 
-            jenkinses.add(new ExecutorJenkins(url, name));
+            jenkinses.add(new ExecutorJenkins(url, name, credential_id));
```

Comment:
When a ``String`` it can be basically anything, what about explicit ``trim()`` and "" -> ``NULL``?